src/bootstrap/bolt.rs

```diff
@@ -49,6 +49,11 @@ pub fn optimize_with_bolt(path: &Path, profile_path: &Path, output_path: &Path)
         .arg("-icf=1")
         // Update DWARF debug info in the final binary
         .arg("-update-debug-sections")
+        // Try to reuse old text segments to reduce binary size
+        .arg("--use-old-text")
+        .arg("--lite=0")
+        .arg("--no-huge-pages")
```

Comment:
I assume the huge page alignment just adds padding, so it should compress fine and not affect download sizes and also shouldn't affect the amount of stuff that gets loaded into memory.

Reply:
I'm not actually sure what this does, as it seems to be undocumented. I'll try without it.